Windows CUDA build is failing

## Describe the bug


>2025-09-23T05:22:30.9338986Z   C:\Users\runneradmin\.cargo\git\checkouts\candle-6740f55d69a3bf41\132643d\candle-flash-attn\kernels\flash_fwd_kernel.h(131): warning #221-D: floating-point value does not fit in required floating-point type
2025-09-23T05:22:30.9339521Z                 if (row < binfo.actual_seqlen_q - m_block * kBlockM && get<1>(tOcO(0, m, 0)) == 0) { gLSE(row) = ((float)(1e+300)); }
2025-09-23T05:22:30.9339707Z                                                                                                                   ^
2025-09-23T05:22:30.9339774Z             detected during:
2025-09-23T05:22:30.9341130Z               instantiation of "void flash::compute_attn_1rowblock<Kernel_traits,Is_dropout,Is_causal,Is_local,Has_alibi,Is_even_MN,Is_even_K,Is_softcap,Return_softmax,Params>(const Params &, int, int, int) [with Kernel_traits=Flash_fwd_kernel_traits<64, 128, 128, 4, false, false, cutlass::bfloat16_t, Flash_kernel_traits<64, 128, 128, 4, cutlass::bfloat16_t>>, Is_dropout=false, Is_causal=true, Is_local=false, Has_alibi=false, Is_even_MN=false, Is_even_K=false, Is_softcap=false, Return_softmax=false, Params=Flash_fwd_params]" at line 1095
2025-09-23T05:22:30.9342821Z               instantiation of "void flash::compute_attn<Kernel_traits,Is_dropout,Is_causal,Is_local,Has_alibi,Is_even_MN,Is_even_K,Is_softcap,Return_softmax,Params>(const Params &) [with Kernel_traits=Flash_fwd_kernel_traits<64, 128, 128, 4, false, false, cutlass::bfloat16_t, Flash_kernel_traits<64, 128, 128, 4, cutlass::bfloat16_t>>, Is_dropout=false, Is_causal=true, Is_local=false, Has_alibi=false, Is_even_MN=false, Is_even_K=false, Is_softcap=false, Return_softmax=false, Params=Flash_fwd_params]" at line 33 of C:\Users\runneradmin\.cargo\git\checkouts\candle-6740f55d69a3bf41\132643d\candle-flash-attn\kernels\flash_fwd_launch_template.h
2025-09-23T05:22:30.9344375Z               instantiation of "void flash_fwd_kernel<Kernel_traits,Is_dropout,Is_causal,Is_local,Has_alibi,Is_even_MN,Is_even_K,Is_softcap,Return_softmax>(Flash_fwd_params) [with Kernel_traits=Flash_fwd_kernel_traits<64, 128, 128, 4, false, false, cutlass::bfloat16_t, Flash_kernel_traits<64, 128, 128, 4, cutlass::bfloat16_t>>, Is_dropout=false, Is_causal=true, Is_local=false, Has_alibi=false, Is_even_MN=false, Is_even_K=false, Is_softcap=false, Return_softmax=false]" at line 66 of C:\Users\runneradmin\.cargo\git\checkouts\candle-6740f55d69a3bf41\132643d\candle-flash-attn\kernels\flash_fwd_launch_template.h
2025-09-23T05:22:30.9345519Z               instantiation of "void run_flash_fwd<Kernel_traits,Is_dropout,Is_causal>(Flash_fwd_params &, cudaStream_t) [with Kernel_traits=Flash_fwd_kernel_traits<64, 128, 128, 4, false, false, cutlass::bfloat16_t, Flash_kernel_traits<64, 128, 128, 4, cutlass::bfloat16_t>>, Is_dropout=false, Is_causal=true]" at line 186 of C:\Users\runneradmin\.cargo\git\checkouts\candle-6740f55d69a3bf41\132643d\candle-flash-attn\kernels\flash_fwd_launch_template.h
2025-09-23T05:22:30.9346260Z               instantiation of "void run_mha_fwd_hdim64<T,Is_causal>(Flash_fwd_params &, cudaStream_t) [with T=cutlass::bfloat16_t, Is_causal=true]" at line 9 of C:\Users\runneradmin\.cargo\git\checkouts\candle-6740f55d69a3bf41\132643d\candle-flash-attn\kernels\flash_fwd_hdim64_bf16_causal_sm80.cu
2025-09-23T05:22:30.9346267Z 
2025-09-23T05:22:30.9346912Z   #$ ptxas -arch=sm_87 -m64 "C:/Users/RUNNER~1/AppData/Local/Temp/tmpxft_00001224_00000000-10_flash_fwd_hdim160_bf16_sm80.ptx"  -o "C:/Users/RUNNER~1/AppData/Local/Temp/tmpxft_00001224_00000000-14_flash_fwd_hdim160_bf16_sm80.sm_87.cubin" 
2025-09-23T05:22:30.9348168Z   #$ fatbinary -64 --cicc-cmdline="-ftz=1 -prec_div=0 -prec_sqrt=0 -fmad=1 " "--image3=kind=elf,sm=87,file=C:/Users/RUNNER~1/AppData/Local/Temp/tmpxft_00001224_00000000-14_flash_fwd_hdim160_bf16_sm80.sm_87.cubin" "--image3=kind=ptx,sm=87,file=C:/Users/RUNNER~1/AppData/Local/Temp/tmpxft_00001224_00000000-10_flash_fwd_hdim160_bf16_sm80.ptx" --embedded-fatbin="C:/Users/RUNNER~1/AppData/Local/Temp/tmpxft_00001224_00000000-7_flash_fwd_hdim160_bf16_sm80.fatbin.c" 
2025-09-23T05:22:30.9348471Z   #$ erase C:/Users/RUNNER~1/AppData/Local/Temp/tmpxft_00001224_00000000-7_flash_fwd_hdim160_bf16_sm80.fatbin
2025-09-23T05:22:30.9349612Z   #$ resource file C:\Users\RUNNER~1\AppData\Local\Temp/tmpxft_00001224_00000000-15.res: [-D__CUDA_ARCH__=870 -D__CUDA_ARCH_LIST__=870 -D__NV_LEGACY_LAUNCH -nologo -c -TP  -DCUDA_DOUBLE_MATH_FUNCTIONS -EHsc -O2 -I"cutlass/include" "-IC:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.4\bin/../include"   "C:\Users\RUNNER~1\AppData\Local\Temp\tmpxft_00001224_00000000-10_flash_fwd_hdim160_bf16_sm80.cudafe1.cpp" ]
2025-09-23T05:22:30.9350290Z   #$ cl.exe /std:c++17 @"C:\Users\RUNNER~1\AppData\Local\Temp/tmpxft_00001224_00000000-15.res" -Fo"D:/a/spiceai/spiceai/bin/spiced/../../target/release/build/candle-flash-attn-51870eafbacc28fd/out/flash_fwd_hdim160_bf16_sm80-b8e226bc00ecbaf1.o" 
2025-09-23T05:22:30.9350472Z   tmpxft_00001224_00000000-10_flash_fwd_hdim160_bf16_sm80.cudafe1.cpp
2025-09-23T05:22:30.9351219Z   C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00001224_00000000-10_flash_fwd_hdim160_bf16_sm80.cudafe1.stub.c(7): error C2062: type 'Flash_fwd_kernel_traits<160,128,64,8,false,false,cutlass::bfloat16_t,Flash_kernel_traits<160,128,64,8,cutlass::bfloat16_t>>' unexpected
2025-09-23T05:22:30.9351677Z   C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00001224_00000000-10_flash_fwd_hdim160_bf16_sm80.cudafe1.stub.c(116): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:30.9352130Z   C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00001224_00000000-10_flash_fwd_hdim160_bf16_sm80.cudafe1.stub.c(124): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:30.9352572Z   C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00001224_00000000-10_flash_fwd_hdim160_bf16_sm80.cudafe1.stub.c(132): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4691533Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(364): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4692077Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(372): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4692545Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(380): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4693009Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(388): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4693448Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(396): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4694227Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(404): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4694679Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(412): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4695120Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(420): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4695632Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(428): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4696072Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(436): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4696521Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(444): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4696953Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(452): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4697387Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(460): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4697830Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(468): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4698262Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(476): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4698694Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(484): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4699137Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(492): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4699621Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(500): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4700094Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(508): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4700535Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(516): error C2894: templates cannot be declared to have 'C' linkage
2025-09-23T05:22:31.4700992Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-10_flash_fwd_hdim64_bf16_sm80.cudafe1.stub.c(583): error C2065: 'fatDeviceText': undeclared identifier
2025-09-23T05:22:31.4701391Z C:\Users\runneradmin\AppData\Local\Temp\tmpxft_00000508_00000000-7_flash_fwd_hdim64_bf16_sm80.fatbin.c(11): fatal error C1075: '{': no matching token found
2025-09-23T05:22:31.4701480Z # --error 0x2 --
2025-09-23T05:22:31.4702757Z #$ fatbinary -64 --cicc-cmdline="-ftz=1 -prec_div=0 -prec_sqrt=0 -fmad=1 " "--image3=kind=elf,sm=87,file=C:/Users/RUNNER<del>1/AppData/Local/Temp/tmpxft_0000079c_00000000-14_flash_fwd_hdim256_bf16_sm80.sm_87.cubin" "--image3=kind=ptx,sm=87,file=C:/Users/RUNNER</del>1/AppData/Local/Temp/tmpxft_0000079c_00000000-10_flash_fwd_hdim256_bf16_sm80.ptx" --embedded-fatbin="C:/Users/RUNNER<del>1/AppData/Local/Temp/tmpxft_0000079c_00000000-7_flash_fwd_hdim256_bf16_sm80.fatbin.c"
2025-09-23T05:22:31.4703029Z #$ erase C:/Users/RUNNER</del>1/AppData/Local/Temp/tmpxft_0000079c_00000000-7_flash_fwd_hdim256_bf16_sm80.fatbin
2025-09-23T05:22:31.4704120Z #$ resource file C:\Users\RUNNER~1\AppData\Local\Temp/tmpxft_0000079c_00000000-15.res: [-D__CUDA_ARCH=870 -D__CUDA_ARCH_LIST__=870 -D__NV_LEGACY_LAUNCH -nologo -c -TP -DCUDA_DOUBLE_MATH_FUNCTIONS -EHsc -O2 -I"cutlass/include" "-IC:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.4\bin/../include" "C:\Users\RUNNER<del>1\AppData\Local\Temp\tmpxft_0000079c_00000000-10_flash_fwd_hdim256_bf16_sm80.cudafe1.cpp" ]
2025-09-23T05:22:31.4704851Z #$ cl.exe /std:c++17 @"C:\Users\RUNNER</del>1\AppData\Local\Temp/tmpxft_0000079c_00000000-15.res" -Fo"D:/a/spiceai/spiceai/bin/spiced/../../target/release/build/candle-flash-attn-51870eafbacc28fd/out/flash_fwd_hdim256_bf16_sm80-21dd0f7dd998e506.o"
2025-09-23T05:22:31.4705010Z tmpxft_0000079c_00000000-10_flash_fwd_hdim256_bf16_sm80.cudafe1.cpp
2025-09-23T05:22:31.4705166Z #$ erase C:\Users\RUNNER~1\AppData\Local\Temp/tmpxft_0000079c_00000000-15.res
2025-09-23T05:22:31.4705172Z
2025-09-23T05:22:31.4705231Z --- stderr
2025-09-23T05:22:31.4705292Z flash_api.cu
2025-09-23T05:22:31.4705357Z flash_fwd_hdim96_bf16_sm80.cu
2025-09-23T05:22:31.4705411Z flash_api.cu
2025-09-23T05:22:31.4705481Z flash_fwd_hdim96_bf16_sm80.cu
2025-09-23T05:22:31.4705550Z flash_fwd_hdim128_fp16_sm80.cu
2025-09-23T05:22:31.4705611Z flash_fwd_hdim128_fp16_sm80.cu
2025-09-23T05:22:31.4705684Z flash_fwd_hdim128_fp16_causal_sm80.cu
2025-09-23T05:22:31.4705761Z flash_fwd_hdim128_fp16_causal_sm80.cu
2025-09-23T05:22:31.4705831Z flash_fwd_hdim160_fp16_sm80.cu
2025-09-23T05:22:31.4705895Z flash_fwd_hdim160_fp16_sm80.cu
2025-09-23T05:22:31.4705966Z flash_fwd_hdim160_fp16_causal_sm80.cu
2025-09-23T05:22:31.4706030Z flash_fwd_hdim160_fp16_causal_sm80.cu
2025-09-23T05:22:31.4706094Z flash_fwd_hdim192_fp16_causal_sm80.cu
2025-09-23T05:22:31.4706157Z flash_fwd_hdim192_fp16_causal_sm80.cu
2025-09-23T05:22:31.4706227Z flash_fwd_hdim192_fp16_sm80.cu
2025-09-23T05:22:31.4706287Z flash_fwd_hdim192_fp16_sm80.cu
2025-09-23T05:22:31.4706351Z flash_fwd_hdim224_fp16_causal_sm80.cu
2025-09-23T05:22:31.4706420Z flash_fwd_hdim224_fp16_causal_sm80.cu
2025-09-23T05:22:31.4706483Z flash_fwd_hdim224_fp16_sm80.cu
2025-09-23T05:22:31.4706542Z flash_fwd_hdim224_fp16_sm80.cu
2025-09-23T05:22:31.4706607Z flash_fwd_hdim256_fp16_causal_sm80.cu
2025-09-23T05:22:31.4706724Z flash_fwd_hdim256_fp16_causal_sm80.cu
2025-09-23T05:22:31.4706793Z flash_fwd_hdim32_fp16_causal_sm80.cu
2025-09-23T05:22:31.4706895Z flash_fwd_hdim32_fp16_causal_sm80.cu
2025-09-23T05:22:31.4706963Z flash_fwd_hdim256_fp16_sm80.cu
2025-09-23T05:22:31.4707027Z flash_fwd_hdim256_fp16_sm80.cu
2025-09-23T05:22:31.4707091Z flash_fwd_hdim64_fp16_causal_sm80.cu
2025-09-23T05:22:31.4707164Z flash_fwd_hdim64_fp16_causal_sm80.cu
2025-09-23T05:22:31.4707227Z flash_fwd_hdim96_fp16_causal_sm80.cu
2025-09-23T05:22:31.4707290Z flash_fwd_hdim96_fp16_causal_sm80.cu
2025-09-23T05:22:31.4707352Z flash_fwd_hdim32_fp16_sm80.cu
2025-09-23T05:22:31.4707418Z flash_fwd_hdim32_fp16_sm80.cu
2025-09-23T05:22:31.4707485Z flash_fwd_hdim128_bf16_causal_sm80.cu
2025-09-23T05:22:31.4707593Z flash_fwd_hdim128_bf16_causal_sm80.cu
2025-09-23T05:22:31.4707663Z flash_fwd_hdim64_fp16_sm80.cu
2025-09-23T05:22:31.4707722Z flash_fwd_hdim64_fp16_sm80.cu
2025-09-23T05:22:31.4707792Z flash_fwd_hdim160_bf16_causal_sm80.cu
2025-09-23T05:22:31.4707857Z flash_fwd_hdim160_bf16_causal_sm80.cu
2025-09-23T05:22:31.4707922Z flash_fwd_hdim96_fp16_sm80.cu
2025-09-23T05:22:31.4707983Z flash_fwd_hdim96_fp16_sm80.cu
2025-09-23T05:22:31.4708050Z flash_fwd_hdim192_bf16_causal_sm80.cu
2025-09-23T05:22:31.4708119Z flash_fwd_hdim192_bf16_causal_sm80.cu
2025-09-23T05:22:31.4708182Z flash_fwd_hdim224_bf16_causal_sm80.cu
2025-09-23T05:22:31.4708249Z flash_fwd_hdim224_bf16_causal_sm80.cu
2025-09-23T05:22:31.4708309Z flash_fwd_hdim128_bf16_sm80.cu
2025-09-23T05:22:31.4708377Z flash_fwd_hdim128_bf16_sm80.cu
2025-09-23T05:22:31.4708441Z flash_fwd_hdim256_bf16_causal_sm80.cu
2025-09-23T05:22:31.4708505Z flash_fwd_hdim256_bf16_causal_sm80.cu
2025-09-23T05:22:31.4708571Z flash_fwd_hdim160_bf16_sm80.cu
2025-09-23T05:22:31.4708633Z flash_fwd_hdim160_bf16_sm80.cu
2025-09-23T05:22:31.4708697Z flash_fwd_hdim32_bf16_causal_sm80.cu
2025-09-23T05:22:31.4708761Z flash_fwd_hdim32_bf16_causal_sm80.cu
2025-09-23T05:22:31.4708835Z flash_fwd_hdim64_bf16_causal_sm80.cu
2025-09-23T05:22:31.4708902Z flash_fwd_hdim64_bf16_causal_sm80.cu
2025-09-23T05:22:31.4708908Z
2025-09-23T05:22:31.4709330Z thread '<unnamed>' panicked at C:\Users\runneradmin.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\bindgen_cuda-0.1.5\src\lib.rs:262:21:
2025-09-23T05:22:31.4710895Z nvcc error while executing compiling: "nvcc" "--gpu-architecture=sm_87" "-c" "-o" "D:\a\spiceai\spiceai\bin\spiced\../../target\release\build\candle-flash-attn-51870eafbacc28fd\out\flash_fwd_hdim160_bf16_sm80-b8e226bc00ecbaf1.o" "--default-stream" "per-thread" "-std=c++17" "-O3" "-U__CUDA_NO_HALF_OPERATORS__" "-U__CUDA_NO_HALF_CONVERSIONS__" "-U__CUDA_NO_HALF2_OPERATORS__" "-U__CUDA_NO_BFLOAT16_CONVERSIONS__" "-Icutlass/include" "--expt-relaxed-constexpr" "--expt-extended-lambda" "--use_fast_math" "--verbose" "-D_USE_MATH_DEFINES" "kernels/flash_fwd_hdim160_bf16_sm80.cu"
2025-09-23T05:22:31.4710900Z
2025-09-23T05:22:31.4710952Z # stdout
2025-09-23T05:22:31.4710956Z
2025-09-23T05:22:31.4710959Z
2025-09-23T05:22:31.4711015Z # stderr
2025-09-23T05:22:31.4711020Z
2025-09-23T05:22:31.4711189Z note: run with RUST_BACKTRACE=1 environment variable to display a backtrace
2025-09-23T05:22:31.4711253Z flash_fwd_hdim224_bf16_sm80.cu
2025-09-23T05:22:31.4711319Z flash_fwd_hdim224_bf16_sm80.cu
2025-09-23T05:22:31.4711389Z flash_fwd_hdim96_bf16_causal_sm80.cu
2025-09-23T05:22:31.4711454Z flash_fwd_hdim96_bf16_causal_sm80.cu
2025-09-23T05:22:31.4711515Z flash_fwd_hdim32_bf16_sm80.cu
2025-09-23T05:22:31.4711583Z flash_fwd_hdim32_bf16_sm80.cu
2025-09-23T05:22:31.4711644Z flash_fwd_hdim256_bf16_sm80.cu
2025-09-23T05:22:31.4711702Z flash_fwd_hdim256_bf16_sm80.cu
2025-09-23T05:22:31.4711768Z flash_fwd_hdim64_bf16_sm80.cu
2025-09-23T05:22:31.4711829Z flash_fwd_hdim64_bf16_sm80.cu
2025-09-23T05:22:31.4711832Z
2025-09-23T05:22:31.4712201Z thread '<unnamed>' panicked at C:\Users\runneradmin.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\bindgen_cuda-0.1.5\src\lib.rs:262:21:
2025-09-23T05:22:31.4713801Z nvcc error while executing compiling: "nvcc" "--gpu-architecture=sm_87" "-c" "-o" "D:\a\spiceai\spiceai\bin\spiced\../../target\release\build\candle-flash-attn-51870eafbacc28fd\out\flash_fwd_hdim64_bf16_sm80-eaa7ce7f57eb7351.o" "--default-stream" "per-thread" "-std=c++17" "-O3" "-U__CUDA_NO_HALF_OPERATORS__" "-U__CUDA_NO_HALF_CONVERSIONS__" "-U__CUDA_NO_HALF2_OPERATORS__" "-U__CUDA_NO_BFLOAT16_CONVERSIONS__" "-Icutlass/include" "--expt-relaxed-constexpr" "--expt-extended-lambda" "--use_fast_math" "--verbose" "-D_USE_MATH_DEFINES" "kernels/flash_fwd_hdim64_bf16_sm80.cu"
2025-09-23T05:22:31.4713847Z
2025-09-23T05:22:31.4713897Z # stdout
2025-09-23T05:22:31.4713901Z
2025-09-23T05:22:31.4713904Z
2025-09-23T05:22:31.4714282Z # stderr
2025-09-23T05:22:31.4714293Z
2025-09-23T05:22:32.1760063Z ##[error]Process completed with exit code 1.
2025-09-23T05:22:32.2239849Z Post job cleanup.

--- a/.github/workflows/build_and_release_cuda.yml
+++ b/.github/workflows/build_and_release_cuda.yml
@@ -106,31 +106,31 @@ jobs:
               },
               {
                 compute_cap: "80",
-                runner: "windows-latest",
+                runner: "windows-latest-8core",
                 target_os: "windows",
                 target_arch: "x86_64"
               },
               {
                 compute_cap: "86",
-                runner: "windows-latest",
+                runner: "windows-latest-8core",
                 target_os: "windows",
                 target_arch: "x86_64"
               },
               {
                 compute_cap: "87",
-                runner: "windows-latest",
+                runner: "windows-latest-8core",
                 target_os: "windows",
                 target_arch: "x86_64"
               },
               {
                 compute_cap: "89",
-                runner: "windows-latest",
+                runner: "windows-latest-8core",
                 target_os: "windows",
                 target_arch: "x86_64"
               },
               {
                 compute_cap: "90",
-                runner: "windows-latest",
+                runner: "windows-latest-8core",
                 target_os: "windows",
                 target_arch: "x86_64"
               }

